apps/ukvi-complaints/translations/src/en/pages.json

```diff
@@ -414,6 +414,10 @@
     "caseworker": {
       "subject": "A new UK Visas and Immigration complaint has been submitted"
     }
+  },
+  "error": {
+    "header": "Sorry, your form has not been sent",
+    "text": "Please try again using the online form"
```

Comment:
Has the copy being approved/reviewed?

Reply:
This is the wording that was used by the designers on the FBIS service so we also used it here @robertdeniszczyc2